Vision/proposal.md

```diff
@@ -0,0 +1,62 @@
+by expanding reach, knowledge, education and services even more broadly.
+
+### The W3C
+
+The World Wide Web Consortium (W3C) was founded as an organization to provide a consistent architecture across the rapid pace of progress in the Web, 
+and to build a common community to support its development. 
+It has become an association where diverse voices from around the world and industries work together to evolve the Web.
+
+To build a better future, the W3C must rise even further to the challenge of improving the Web's fundamental integrity, 
+while continuing to expand the Web’s scope and reach. 
+We must embody in our work the core values of the Web itself, 
+by supporting the Ethical Web Principles with Operational Principles of the W3C.
+
+The W3C’s impact will be clearly demonstrated by how it leads the Web forward: 
+by being inclusive, principled, and continually striving to make the Web better through these principles and the Ethical Web Principles. 
+As the Ethical Web Principles state, “The web should empower an equitable, informed and interconnected society.”
+
+## Operational Principles of the W3C
+
+* The W3C intentionally involves stakeholders from end to end in building the Web: developers, content creators, and end users.
+  Our work will not be dominated by any person, company, or interest group, but uses consensus from the community
+  to put the needs of users first: above authors, publishers, and implementers.
+* We believe in the diversity of the Web, and its inclusion of participants
+  from different geographical locations, cultures, languages, disabilities, gender identities, industries, user groups, organizational sizes, and more.
+  In addition to ensuring the W3C serves the needs of the entire Web user base,
+  we must also strive to broaden diversity and inclusion for our own participants.
+* We strive to create standards with an ethical intent to improve equity of access and participation on the Web.
+  We will ensure the web platform meets broad goals using consistent horizontal review
+  to ensure accessibility, internationalization, security, privacy, and sustainability.
+* We believe wide and interoperable implementation is a requirement for standards.
+  To ensure reliable interoperability, we require multiple implementations and open test suites for our standards.
+* The Web will continue to expand in user base, global reach, and technical breadth.
+  We are committed to encouraging incubation in new areas, collaborating on new innovations across our community.
+* We exclusively create open royalty-free standards.
+  Our process is rooted in open development, strong royalty-free patent policy and open copyright licenses.
+* The W3C is committed to collaborative relationships with other Internet and Web standards organizations,
+  including building and maintaining respected relationships with governments and businesses for providing credible advice.
```

Comment:
"including" feels off since governments and businesses aren't Internet and Web standards organizations.
```suggestion
  and building and maintaining respected relationships with governments and businesses for providing credible advice.
```